assignment/api/src/resources/assignment_ca.properties

```diff
@@ -269,7 +269,8 @@ gen.hideduedate=Amaga als estudiants la data de lliurament
 grad2=La qualificaci\u00f3 introdu\u00efda \u2019{0}\u2019 \u00e9s m\u00e9s gran que la m\u00e0xima permesa \u2019{1}\u2019. Corregiu-ho o  feu clic altre cop sobre el bot\u00f3 per desar la qualificaci\u00f3 introdu\u00efda \u2019{0}\u2019.
 
 grad3=Qualificada
-
+grade.title=Qualificacions
+grade.delimiter=,
```

Comment:
Never put delimiters into translation files.

Check the formattedText.getDecimalSeparator() method.